doc/direction.md

```diff
@@ -0,0 +1,91 @@
+# Minetest Direction Document
+
+---------------------------
+
+## 0. Preface
+
+### 0.1. Introduction
+
+This document attempts to set out Minetest's philosophy, aims, and objectives.
+This should be considered as a non-binding summary.
+
+### 0.2. Definitions
+
+* `Minetest` is the game engine.
+* `Minetest Game` is the default game which comes with the engine, and is not covered by this document.
+* The `Minetest Project` refers to the combination of the above, the development team, and supporting tools.
+* `Content` refers to any installable add-ons, such as Mods, Games, and Texture Packs.
+* Also see the [Terminology](https://dev.minetest.net/Terminology) page on the Minetest dev wiki.
+
+### 0.3. Other documents
+
+In order to understand this document, it's important to have context.
+
+celeron55 has published his opinions on the development of Minetest in these places:
+
+  * What is Minetest? - http://c55.me/blog/?p=1491
+  * Roadmap - https://forum.minetest.net/viewtopic.php?t=9177
+  * A clear mission statement for Minetest is missing - https://github.com/minetest/minetest/issues/3476#issuecomment-167399287
+
+Also see the roadmaps and manifestos of core developers on the forums: https://forum.minetest.net/viewforum.php?f=7
+
+---------------------------
+
+## 1. Mission Statement
+
+Minetest provides both a game engine, which allows the creation of voxel games using Lua, and
+a content distribution platform. Players can connect to servers which provide their own gameplay,
+and easily install content through the Minetest client. The goal is to provide an easy to learn and use,
+but still powerful system which allows to easily and rapidly create, extend and modify voxel games.
+
+## 2. Aims
+
+What should changes to the engine take into mind?
+
+1. **Stability and performance**.
+		Mobile and low-end devices should not be excluded from essential graphical
+		and gameplay elements.
+2. **Keep the engine as universal as possible**.
+		It's worth comparing everything that goes into the engine for how well
+		it suits a gravitation-less space game without ground surface.
+		If everything is either disableable or suitable for such, then it gives
+		much more freedom for people to experiment and create interesting things,
+		even in ground-based worlds.
+3. **Keep the engine lightweight**.
+		Support a large amount of gameplay by pushing code and functionality out
+		to Lua, where possible, rather than having a complex core.
+		The exception to this is when APIs are needed to increase performance.
+		Keeping the core as small as possible makes it easier to test and more flexible.
```

Comment:
This section feels like it belongs in the sections preceding and following it. Keeping features in Lua sounds like a flexibility problem and not a performance one, and thus belongs in the sections referencing a gravitation-less space game. Similarly, stating that it's easier to maintain sounds like it belongs in the "Prefer simplicity" section as that's about code simplicity.

Basically, what is this section saying that isn't already, or couldn't just be stated in the other two sections?